### ISSUE TITLE
Error when merging datacheck h5 files due to mistyped muon info

### ISSUE THREAD

**Issue by morcuended** [cta-observatory/cta-lstchain#414]
For some of the runs I found out that an error related to muon info raises when running `lstchain_check_dl1`

For example:
```
$ lstchain_check_dl1 --input-file="/fefs/aswg/data/real/DL1/20200125/v0.5.1_v03/datacheck_dl1_LST-1.Run01863.*.h5"
```

```python
Traceback (most recent call last):
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/table/operations.py", line 858, in _vstack
    out[out_name] = col_cls.info.new_like(cols, n_rows, metadata_conflicts, out_name)
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/table/column.py", line 174, in new_like
    ('meta', 'unit', 'format', 'description'))
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/utils/data_info.py", line 642, in merge_cols_attributes
    out['dtype'] = metadata.common_dtype(cols)
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/utils/metadata.py", line 64, in common_dtype
    raise tme
astropy.utils.metadata.MergeConflictError: Arrays have incompatible types ['float64', 'bool']

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/bin/lstchain_check_dl1", line 8, in <module>
    sys.exit(main())
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/lstchain/scripts/lstchain_check_dl1.py", line 102, in main
    plot_datacheck(filenames, args.output_dir, args.muons_dir)
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/lstchain/datachecks/dl1_checker.py", line 819, in plot_datacheck
    muons_table = vstack([muons_table, t])
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/table/operations.py", line 292, in vstack
    out = _vstack(tables, join_type, col_name_map, metadata_conflicts)
  File "/fefs/aswg/software/virtual_env/anaconda3/envs/osa/lib/python3.7/site-packages/astropy/table/operations.py", line 863, in _vstack
    .format(out_name, err._incompat_types))
astropy.table.np_utils.TableMergeError: The 'good_ring' columns have incompatible types: ['float64', 'bool']
```

Actually the merged h5 and pdf files are produced but (at least) the pdf does not contain the muon check plots.

**Comment by moralejo** [comment]
Most muon files for that run have no events, a few have one event. In the former, all column types are float64, including good_ring which should be boolean. When trying to merge them astropy complains. The underlying reason for the missing muons may be image analysis is completely wrong for some reason (noisy pixels?) and then no event passes the ring candidate cuts. Here I will make a check for the number of events in the files, and display a more user-friendly message. But the underlying problem has to be investigated.

**Comment by morcuended** [comment]
A few months ago I tried to go through all the ELOGs but probably did not spot that garbage run. Looking at the spreadsheet I shared with all of you I indeed noted that such a run might not be good. I will remove then this run from the list. I will check whether this has happened for some other runs with the same missing-muon-events problem.

**Comment by moralejo** [comment]
Should be solved by #417

**Comment by rlopezcoto** [comment]
Solved via #417
